mantidproject/mantid
# Update ISIS Sans Documentation

**Description of work.**
Starts pulling in some of the docs from [the ISIS SANS Wiki](https://www.mantidproject.org/ISIS_SANS) into .rst files

- Split ISIS SANS Interfaces docs into their own folder
- Chunk down each page
- Update all the images, and organise them into a folder
- Some text fixing, however maybe 1/4 was proofed - the rest will be do as the docs are updated
- Fixed some other minor docs issues (such as pages not appearing in categories)

There is some additional work to port from the wiki, but these are more edge case pages which each need to be proof read before adding (as some don't apply).

**To test:**
Note: I **don't** expect people to proof read this, it's some minor grammar fixes and splitting / porting of existing text from the wiki and internal docs.

- Check that the pages build without warnings
- Go to docs/interfaces/ISIS SANS
- Check do all the links work and do the images correctly load

Fixes #22816. 


*This does not require release notes* because the documentation was already split between the two, and this is a copy + paste + reformat exercise

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
